### ISSUE TITLE
Incorrect data was entered during structure field completion

### ISSUE THREAD

**Issue by LuoZhongYao** [Shougo/deoplete-lsp#27]
```c
struct a {
	int b;
};

int main(void)
{
	struct a a;
        return a.   return a.b;
}

```

Input `return a.` to bring up `b Filed[lsp] int`, use `<c-p>` to select `b Filed[lsp] int`, the actual input is `return a.b`

**Comment by Shougo** [comment]
Please upload nvim-lsp configuration to reproduce the problem.
If not, I will close the issue.

**Comment by LuoZhongYao** [comment]
```vim
runtime! debian.vim                                                                                                                                             
call plug#begin(stdpath('config') . '/plugged')                                  
Plug 'Shougo/deoplete.nvim', { 'do': ':UpdateRemotePlugins' }                    
Plug 'Shougo/deoplete-lsp'                                                       
Plug 'lifepillar/vim-solarized8'                                                 
Plug 'godlygeek/tabular'                                                         
Plug 'neovim/nvim-lsp'                                                           
call plug#end()                                                                  
                                                                                 
let g:deoplete#enable_at_startup = 1                                             
" lua require'nvim_lsp'.bashls.setup{}                                           
lua require'nvim_lsp'.cmake.setup{}                                              
lua require'nvim_lsp'.clangd.setup{}  
```

```shell
NVIM v0.5.0-579-g1920ba4b5
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc-5 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/travis/build/neovim/bot-ci/build/neovim/build/config -I/home/travis/build/neovim/bot-ci/build/neovim/src -I/home/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/build/src/nvim/auto -I/home/travis/build/neovim/bot-ci/build/neovim/build/include
Compiled by travis@travis-job-5774ea5a-d34f-44bf-8fb8-9369d08e2fa8

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/share/nvim"

Run :checkhealth for more info

```

**Comment by Shougo** [comment]
Reproduced.

**Comment by Shougo** [comment]
The patch https://github.com/Shougo/deoplete-lsp/pull/25 seems broken.

**Comment by Shougo** [comment]
@kdheepak Please test the case.

**Comment by Shougo** [comment]
I need to fix #25 patch.  Please wait.
